package.json

```diff
@@ -889,72 +819,6 @@
             {
                 "language": "javascript"
             }
-        ],
-        "debuggers": [
-            {
-                "type": "jsLogpoints",
-                "label": "Azure App Service LogPoints",
-                "program": "./out/src/diagnostics/LogPointsDebugAdapter.js",
-                "runtime": "node",
-                "configurationAttributes": {
-                    "launch": {
-                        "required": [
-                            "sessionId",
-                            "debugId"
-                        ],
-                        "properties": {
-                            "trace": {
-                                "type": "boolean",
-                                "description": "Enable logging of the Debug Adapter Protocol.",
-                                "default": true
-                            },
-                            "siteName": {
-                                "type": "string",
-                                "description": "A URL prefix for the azure scm url.",
-                                "default": ""
-                            },
-                            "publishCredentialUsername": {
-                                "type": "string",
-                                "description": "publishingUserName",
-                                "default": ""
-                            },
-                            "publishCredentialPassword": {
-                                "type": "string",
-                                "description": "publishingPassword",
-                                "default": ""
-                            },
-                            "instanceId": {
-                                "type": "string",
-                                "description": "ARRAffinity",
-                                "default": ""
-                            },
-                            "sessionId": {
-                                "type": "string",
-                                "description": "sessionId",
-                                "default": ""
-                            },
-                            "debugId": {
-                                "type": "string",
-                                "description": "debugId",
-                                "default": ""
-                            }
-                        }
-                    }
-                },
-                "initialConfigurations": [
-                    {
-                        "type": "mock",
-                        "name": "Azure App Service LogPoints",
-                        "request": "attach",
-                        "program": "readme.md",
-                        "trace": true,
-                        "hostname": "",
-                        "instanceId": "",
-                        "sessionId": "",
-                        "debugId": ""
-                    }
-                ]
-            }
         ]
     },
```

Comment:
Please double check the `dependencies` and `devDependencies` to see if there are any that can be removed. I believe at least `vscode-debugadapter` and `vscode-debugprotocol` could be removed